apps/parsley/src/components/LogRow/SectionStatusIcon/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { palette } from "@leafygreen-ui/palette";
+import Icon from "components/Icon";
+import { SectionStatus } from "constants/logs";
+
+const { gray, red } = palette;
+
+interface Props {
+  status: SectionStatus;
+}
+
+export const SectionStatusIcon: React.FC<Props> = ({ status }) => {
+  const fill = status === SectionStatus.Pass ? gray.dark1 : red.base;
```

Comment:
[DEVPROD-10022](https://jira.mongodb.org/browse/DEVPROD-10022) mentions adding a green fill for passing, should we add a green fill here?